source/classification1.md

```diff
@@ -5,15 +5,14 @@ jupytext:
     extension: .md
     format_name: myst
     format_version: 0.13
-    jupytext_version: 1.13.7
+    jupytext_version: 1.14.7
 kernelspec:
   display_name: Python 3 (ipykernel)
   language: python
   name: python3
 ---
 
 ```{code-cell} ipython3
-:tags: [remove-cell]
```

Comment:
Definitely should still be removing this cell

Reply:
Yeah sorry that was me playing around, added it back